### ISSUE TITLE
Incorrect handling of newlines in JSON

### ISSUE THREAD

**Issue by moshez** [moshez/rfc7464#1]
As I understand the standard,

```
<RS>{
"hello": "world"
}
```
Is a valid JSON-SEQ, which should parse to one element, `{"hello": "world"}`. Currently, the code assumes elements are all contained in one line.

**Comment by AstraLuma** [comment]
Indeed, per https://tools.ietf.org/html/rfc7464#section-2.1 the newline should not be part of the parsing process.

Roughly, parsing is a streaming version of `(json.loads(possible_json.decode('utf-8') for possible_json in data.split(b'\x1E'))`
